src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -505,8 +481,11 @@ bool ansi_c_entry_point(
   else
   {
     // produce nondet arguments
-    call_main.arguments()=
-      build_function_environment(parameters, init_code, symbol_table);
+    call_main.arguments()=build_function_environment(
```

Comment:
Ah, apologies for being unclear: *also* wrap after the `=` (and then indent the arguments by two additional spaces).